third_party/move/move-model/src/ast.rs

```diff
@@ -1853,18 +1878,14 @@ impl ExpRewriterFunctions for ExpRewriter<'_> {
 struct LoopNestRewriter {
     loop_depth: usize,
     delta: isize,
+    rewrite: fn(Exp, usize, usize, bool, isize) -> Exp, // args: expression, current loop depth, nest of the loop-cont, whether it is a continue, delta to be adjusted on the nest
```

Comment:
nit: make the comment a field comment with `///` appearing in the line before this field